JKH/Day3/자릿수 더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(n)
+{
+  let pow = 0;
+  let answer = 0;
+
+  while(1) {
+    if (n / Math.pow(10, pow) < 1) break;
+    pow++;
+  }
+
+  for (pow; pow >= 0;) {
+    --pow;
```

Comment:
```suggestion
  for (let pow = String(n).length - 1; pow >= 0; pow--) {
```
pow를 외부에서 선언하지 않고 for문 안에서 바로 시작값으로 지정하여 사용하면 더 가독성이 높아질 것 같습니다.

Reply:
코드를 다 작성하고 나서 미묘한 저 --pow가 정말 곤란했는데요, 상단에서 작성해주신 것처럼 좋은 방법 제시해주셔서 정말 고맙습니다😊